### PR TITLE
refactor: make_empty_graph() is now fully auto-generated

### PR DESCRIPTION
**Pull request by szhorvat**
Related to #1140

`make_empty_graph()` is now fully auto-generated. The custom implementation is no longer needed after #1051.

This PR conflicts with the first commit from the draft PR #1044.

### CODE REVIEW

**Comment by krlmlr** [comment]
Why is this urgent?

**Comment by szhorvat** [comment]
If you don't want to merge it now your can put it on hold. This was part of verifying that scalar conversion checks are working as intended and there are no crashes

**Comment by szhorvat** [comment]
I should mention that this also catches errors that the previous version did not (passing a longer vector where a scalar is expected). I don't see any disadvantage to merging this, quite the contrary.

**Comment by szhorvat** [comment]
Is there any reason to delay merging this? Instead of saying it conflicts with the first commit in #1044, I should have said that it _replaces_ that commit. This is not a breaking change, just simplifies codes and uses standard error reporting instead of a custom version.

**Comment by krlmlr** [comment]
We can merge right after the 2.0.0 release, hopefully next week, unless the revdepchecks show anything major.

**Comment by krlmlr** [comment]
Do we never intend to expose or even use `make_empty_attrs()` ? Should we add a comment to explain the reasoning, like you did for other low-level functions earlier?

I still need to run revdepchecks on this.

**Comment by szhorvat** [comment]
> Do we never intend to expose or even use `make_empty_attrs()` ?

No



> Should we add a comment to explain the reasoning, like you did for other low-level functions earlier?

Done



> I still need to run revdepchecks on this.

Risk of breakage is very low, but always a good practice to run revdepchecks

File: tools/stimulus/functions-R.yaml [hunk 1]
```diff
@@ -11,7 +11,10 @@
 #######################################
 
 igraph_empty:
-    IGNORE: RR
+
+# Not needed in R, we use igraph_emtpy() instead
+igraph_empty_attrs:
+    IGNORE: RR, RC
```
Comment (krlmlr) [inline tools/stimulus/functions-R.yaml:17]: Thanks. Why has the order changed now? I remember aligning this file with `functions.yaml` from the C core, I'd like both to be in the same order. No need to be religious about it, but let's keep the drift here to a minimum moving forward.

**Comment by krlmlr** [comment]
Thanks!

File: tools/stimulus/functions-R.yaml (hunk 1 above)
Comment (szhorvat) [reply tools/stimulus/functions-R.yaml:17]: I changed the order to be able to see both related functions next to it. If you're bothered by it, I'll change it in the C core too.

Comment (krlmlr) [reply tools/stimulus/functions-R.yaml:17]: Thanks. Changing in the C core would be great, if it's idiomatic there.

Comment (szhorvat) [reply tools/stimulus/functions-R.yaml:17]: Done.

**Pull request merged**